core/run/src/tests.rs

```diff
@@ -177,6 +225,13 @@ fn check_hashes(chain: &str, name: &str, expected_str: &str, actual: H256) {
     );
 }
 
+fn assert_metadata(metadata_0: Metadata, metadata_1: Metadata) {
```

Comment:
```suggestion
fn assert_metadata(metadata_a: Metadata, metadata_b: Metadata) {
```

Reply:
suggestion: To avoid confusion with 
https://github.com/axonweb3/axon/blob/f000c0f2651dc6a123e4d7d84f9262cb292f3ef1/core/run/src/tests.rs#L215-L217

Reply:
The `propose_count` field in `Metadata` struct should not be compared. So the `assert_metadata` function compare the left and right field by field.